### PR TITLE
Create Muslim_Aid.org.xml

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**

### CODE REVIEW

File: src/chrome/content/rules/Muslim_Aid.org.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ruleset name="Muslim Aid.org">
+	<target host="muslimaid.org" />
+	<target host="www.muslimaid.org" />
+	<target host="qurbani.muslimaid.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/Muslim_Aid.org.xml]: `www.ramadan.muslimaid.org`

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI I tried it earlier  but it didn't work even with HTTP alone, should I include it nevertheless?

**Comment by J0WI** [comment]
I can confirm that `www.ramadan.muslimaid.org` does not exist, so you can ignore this. But `ramadan.muslimaid.org` seems to be there.

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added

**Pull request merged**